pixeltable/exec/expr_eval/schedulers.py

```diff
@@ -76,7 +76,7 @@ def _set_pool_info(self) -> None:
         assert hasattr(self.pool_info, 'get_request_resources')
         sig = inspect.signature(self.pool_info.get_request_resources)
         self.get_request_resources_param_names = [p.name for p in sig.parameters.values()]
-        self.est_usage = {r: 0 for r in self._resources}
+        self.est_usage = dict.fromkeys(self._resources, 0)
```

Comment:
I'm finding this harder to read. I'd have to look up with that means, which wasn't the case before.

Reply:
See preceding comments, I just don't think "Marcel finds it harder to read" is a good justification for disabling a linting rule. The fact that you'd have to look it up speaks to your individual fluency with this Python pattern; it doesn't imply it's poor practice to use it.

Perhaps to someone else who's more familiar with this pattern, the conventional form is actually easier to read. (Personally, I rather prefer it.) I'd surmise there's a performance benefit too, since no generator is involved (though again I'm not an expert on Python performance, but I do default to trusting the linter).